decidim-members.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'decidim-core', '~> 0.15', '>= 0.15.0'
   s.add_dependency 'pg_search', '~> 2.1.4'
-  s.add_dependency 'rails', '~> 5.2'
+  s.add_dependency 'rails', '~> 5.2.2.1'
```

Comment:
Add a comment explaining why we add this and when it can be removed please.

Reply:
Done